src/main/java/gregtechfoodoption/item/GTFOMetaItem.java

```diff
@@ -434,6 +459,43 @@ public void registerSubItems() {
         HOT_RABBIT_STEW = addItem(133, "food.rabbit_stew.hot").addComponents(new GTFOFoodStats(9, 0.9f, false, false, new ItemStack(Items.BOWL))
                 .setEatingDuration(60));
 
+        SKEWER = addItem(135, "component.skewer");
+        KEBAB_KUBIDEH = addItem(136, "food.kebab.kubide").addComponents(new GTFOFoodStats(3, 0.2f, false, false, SKEWER.getStackForm(1))
```

Comment:
Consider adding negative effects to uncooked foods, or make them not consumable.